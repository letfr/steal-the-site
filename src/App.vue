<template>
  <div id="app">
  <link rel="stylesheet"
  href="https://use.fontawesome.com/releases/v5.2.0/css/all.css"
  integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ"
  crossorigin="anonymous" />
    <NavBar class="navbar" />
    <router-view />
    <footer>
      FEITO COM ❤️ PELA LOOK STEALER LETICIA FARIA
    </footer>
  </div>
</template>

<script>
import NavBar from '@/components/NavBar.vue';

export default {
  components: { NavBar },
};
</script>

<style lang="scss">
@import '../public/reset.css';
@import 'assets/css/variables.css';
@import 'assets/css/global.css';

.app {
  height: 100vh;
}

footer {
  color: #fff;
  font-family: 'Open Sans', sans-serif;
  font-weight: 600;
  padding: 10px;
  background-color: #000;
  text-align: center;
}

@media (max-width: 600px) {
  footer {
    font-size: 12px;
  }
}
</style>
