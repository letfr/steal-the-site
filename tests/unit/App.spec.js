import { shallowMount, createLocalVue } from '@vue/test-utils';
import App from '@/App.vue';

describe('App.vue', () => {
  let components;
  let wrapper;

  beforeAll(() => {
    const localVue = createLocalVue();

    components = {
      NavBar: jest.fn(),
      'router-view': jest.fn(),
    };

    wrapper = shallowMount(App, {
      components,
      localVue,
    });
  });

  it('Should match snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
