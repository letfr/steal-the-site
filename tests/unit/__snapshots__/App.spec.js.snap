// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue Should match snapshot 1`] = `
<div id="app">
  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.2.0/css/all.css" integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ" crossorigin="anonymous">
  <!---->
  <!---->
  <footer>
    FEITO COM ❤️ PELA LOOK STEALER LETICIA FARIA
  </footer>
</div>
`;
